### PR TITLE
fix: Removed resize option from bio editor

### LINKED ISSUE #11378: [CAL-3954] Hard to Focus on Text Area to enter User About description
### Is your proposal related to a problem?

It is very hard to focus on the text area to enter the description in About section. And also the formatting options are just above the text area. So we i can't able to find the focus of text area.

### Describe the solution you'd like

Try to increase the text area height so that we can easily focus to text area. And also shift the formatting options to right of text area .

<sub>[CAL-3954](https://linear.app/calcom/issue/CAL-3954/hard-to-focus-on-text-area-to-enter-user-about-description)</sub>

### PR DESCRIPTION
**Pull request by thepradipvc**
## What does this PR do?

I removed the resize option from the bio editor to avoid unexpected issues due to user interaction.

Fixes #11378

- [x] Bug fix (non-breaking change which fixes an issue)

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by up1512001** [comment]
You can add fix: prefix in your PR name so it can pass the Validate PRs check.

**Review by sean-brydon — CHANGES_REQUESTED** [review]
I dont think we should remove the resize - we should just increase the default min-height for this component. I'd prefer to do this on a usecase basis also instead of all editors. 

Please also uncommit yarn.lock

git checkout origin/main -- yarn.lock

File: packages/ui/components/editor/stylesEditor.css [hunk 1]
```diff
@@ -33,7 +33,6 @@
   border-bottom-left-radius: 6px;
   border-bottom-right-radius: 6px;
   overflow: scroll;
-  resize: vertical;
```
Comment (Udit-takkar) [inline packages/ui/components/editor/stylesEditor.css:36]: why did you remove this?

Comment (ChaudharyPradip) [reply packages/ui/components/editor/stylesEditor.css:36]: See, I don't think there is a need to increase the min-height for the editor because it automatically gets bigger when the content gets longer.

However, if a user uses the resize controller to resize the height of the editor. It will then stop adjusting to the size of the content. It will no longer adapt to the content length but if we remove the option to resize the editor it will always adapt to the content.

But, I'm not aware of any other place where the editor size might need to be adjusted by the user (I don't think it would be a necessity). So if that's the case then please ignore this change.

**Comment by RavishankarMahto** [comment]
**I can solve this [issue.**

**Comment by RavishankarMahto** [comment]
I can solve this issue.
